resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh

```diff
@@ -46,12 +46,6 @@ shift 1
 SPARK_CLASSPATH="$SPARK_CLASSPATH:${SPARK_HOME}/jars/*"
 env | grep SPARK_JAVA_OPT_ | sort -t_ -k4 -n | sed 's/[^=]*=\(.*\)/\1/g' > /tmp/java_opts.txt
 readarray -t SPARK_JAVA_OPTS < /tmp/java_opts.txt
```

Comment:
In general I would like to refactor the name here of SPARK_JAVA_OPTS, its a bit misleading as this property used to exist in Spark and was removed: https://issues.apache.org/jira/browse/SPARK-14453

Reply:
it would be confusing -- this is local though right, not exported?

Reply:
@felixcheung yes it is local, but for users not to make assumptions it would be good to rename it at some point.